apps/ssr-tests-v9/src/utils/visitPage.ts

```diff
@@ -14,9 +15,21 @@ export async function visitPage(browser: Browser, url: string) {
 
   page.on('console', message => {
     if (message.type() === 'error') {
+      const messageContent = message.text();
+
+      // Ignoring 'aria-description' warning from react 17 as it's a valid prop
+      // https://github.com/facebook/react/issues/21035
+      if (
+        messageContent.startsWith('Warning: Invalid aria prop %s on <%s> tag.') &&
```

Comment:
**suggestion:** cant we use the whole string instead of having 2 compare methods ?

```ts
React.version.startsWith('17') && messageContent.includes('Warning: Invalid aria prop %s on <%s> tag. For details, see https://reactjs.org/link/invalid-aria-props%s`aria-description`div
    at div')
```

**suggestion 2:** also it's usually good idea to encapsulate these kind of verbose logics into small function

Reply:
thanks for the review, moved to helper message and added the entire content. Personally didn't like the link there so I separated it but nit :)